rgw/COSBench/cosbench.py

```diff
@@ -0,0 +1,269 @@
+"""Module to trigger COS Bench workloads."""
+import argparse
+import json
+import logging
+import subprocess
+import sys
+from os.path import dirname, join
+from tempfile import NamedTemporaryFile
+from typing import Dict, Optional, Tuple
+
+from jinja2 import Template
+from yaml import safe_load
+
+APP_DIR = "/opt/cosbench"
+LOG = logging.getLogger(__name__)
+
+
+def exec_command(cmd: str) -> subprocess.CompletedProcess:
+    """Execute the given command.
+
+    Args:
+        cmd (str):  The command that needs to be executed.
+
+    Returns:
+
+    """
+    LOG.debug("Executing command: %s", cmd)
+    return subprocess.run([cmd], shell=True, check=True, stdout=subprocess.PIPE)
+
+
+def get_user_info() -> Dict:
+    """Returns cosbench01 user info available in realm."""
+    try:
+        proc = exec_command(cmd="radosgw-admin user info --uid cosbench01")
```

Comment:
Can we have support for passing custom users apart from default ones?

Reply:
Having tenanted users would give us better coverage. We can take it later.

Reply:
It should be possible, can I pick this up in a different PR?

Reply:
Yup, we can have this addressed in a different PR.